cmd/internal/applicationsnapshot/source_test.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright © 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package applicationsnapshot
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_NewSchemaSource(t *testing.T) {
+	source := NewSchemaSource()
+	assert.IsType(t, schemaSource{}, source)
+}
```

Comment:
We want to test behavior that is not enforced by the compiler, but this helps with test coverage...

Reply:
Yeah, that's a coverage test.